### PR TITLE
[RelNotes] Add all known contributors with >1 commit (NFC). [skip-ci]

### PR DESCRIPTION
**Pull request by Axel-Naumann**
Please lmk if you want affiliations changed. FYI, @HDembinski @SimeonEhrig @matthewfeickert @olifre @sayang14 @weliveindetail @ShamrockLee @Moelf @mrexodia @tapaswenipathak @dkpoult @mphschmitt @Neel-Shah-29 @petrstepanov @saisoma123 @andriish @fwinkl @junaire 

And thank you 💐 for your contributions to ROOT v6.28 - much appreciated!

### CODE REVIEW

File: README/ReleaseNotes/v628/index.md [hunk 1]
```diff
@@ -18,44 +18,67 @@ The following people have contributed to this new version:
...
  Enrico Guiraud, CERN/SFT,\
  Stephan Hageboeck, CERN/IT,\
  Jonas Hahnfeld, CERN/SFT,\
  Ahmat Mahamat Hamdan, CERN/SFT,\
  Fernando Hueso-González, University of Valencia,\
+ Pawan Johnson, Indian Institute of Technology Kharagpur,\
  Subham Jyoti, ITER Bhubaneswar,\
+ Ivan Kabadzhov, CERN/SFT,\
+ Giovanna Lazzari Miotto, CERN/SFT,\
+ Shamrock Lee,\
```
Comment (ShamrockLee) [inline README/ReleaseNotes/v628/index.md:47]: My legal name is Yueh-Shun Li (李岳瞬 in Mandarin). I'm a graduate student in National Central University (in Taiwan) working on HEP data analysis using CMS dataset.

```suggestion
 Shamrock Lee,\
 Yueh-Shun Li, National Central University,\
```

**Comment by Axel-Naumann** [comment]
You know what, let me just merge it; let's do iterations if something is not perfect!

**Review by ShamrockLee — CHANGES_REQUESTED** [review]
Thanks for including me in the contributor list! It's my honor to participate in such a great project!

File: README/ReleaseNotes/v628/index.md (hunk 1 above)
Comment (Axel-Naumann) [reply README/ReleaseNotes/v628/index.md:47]: > My legal name is

You can freely choose the name you want to see here; I take your comment that you want to see "Yueh-Shun Li, National Central University"; I'll add "Taiwan/CMS", i.e.
```suggestion
 Yueh-Shun Li, National Central U Taiwan/CMS,\
```
OK?

Comment (ShamrockLee) [reply README/ReleaseNotes/v628/index.md:47]: Sure. Thank you!

Comment (Axel-Naumann) [reply README/ReleaseNotes/v628/index.md:47]: Thanks, that's now 58d14187f09e2f0a6a0fd08174dec3ad5e3a6c12

**Comment by ShamrockLee** [comment]
Just found that the above change is made in the 6.26 release note on the `v6-28-00-patches` branch, and the 6.28 release note is still waiting for the update.

**Comment by Axel-Naumann** [comment]
Thanks @ShamrockLee fixed!

**Pull request merged**